src/pkg/control/options.go

```diff
@@ -62,6 +62,10 @@ type Toggles struct {
 	// DisableIncrementals prevents backups from using incremental lookups,
 	// forcing a new, complete backup of all data regardless of prior state.
 	DisableIncrementals bool `json:"exchangeIncrementals,omitempty"`
+	// DisableAssistCaching disables finding cached items in previous failed
+	// backups (i.e. kopia-assisted incrementals). Data dedupe will still occur
+	// since that is based on content hashes.
+	DisableAssistCaching bool `json:"disableAssistCaching,omitempty"`
```

Comment:
happy to take suggestions on better names if folks have ideas

Reply:
I mentioned this in the CLI PR, but I think `caching` is overused and difficult to correctly interpret.  Granted, having the space to call it `assistCaching` helps.  But I still think it's better avoided.  `DisableStorageAssist`, maybe.  Or, if you want to keep with the suggestion in the other PR, `ForceFullBackup`.